broadinstitute/pearl
# AR-68 HeartHive skeleton project

This creates a skeleton project for HeartHive.  It also includes a bunch of cleanups and enhancements of the participant view to support HeartHive minimal functionality.  Note that we are still planning to have HeartHive keep its SquareSpace page and only use NewProd for the logged-in hub, but cleaning up the landing page and giving a placeholder will make development and testing easier (as well as giving fallback options).

TO TEST:
1. nuke/reset DB, restart app servers
2. run `scripts/populate_setup.sh` and `scripts/populate_portal.sh hearthive`
3. go to `http://sandbox.hearthive.localhost:3001/`  and confirm you see this screen
![image](https://user-images.githubusercontent.com/2800795/211092774-9ac6ce3d-3b83-4b8a-8d15-9bdea5a90f80.png). 
 The "register" button will not work yet as the portal-only join functionality has not yet been built
4. log into the admin tool, and confirm that two studies are visible in the portal dashboard
![image](https://user-images.githubusercontent.com/2800795/211092902-9879c5a3-3d76-43af-8cb9-09a2e84c1878.png)
